packages/devtools_app/lib/src/debugger/codeview.dart

```diff
@@ -166,6 +166,27 @@ class _CodeViewState extends State<CodeView>
 
   @override
   Widget build(BuildContext context) {
+    return ValueListenableBuilder(
+        valueListenable: widget.controller.showFileOpener,
+        builder: (context, showFileOpener, _) {
+          return Stack(
```

Comment:
this will result in two Stack widgets. One here and one as part of buildCodeviewContent. Instead of adding the buildCodeviewContent helper, leave this build method as it was, and you can add your ValueListenableBuilder and extra Positioned widget directly to the old build method.

We should also make sure that both the file opener and the search in file fields cannot be showing at the same time. Opening one of the two while the other is open should automatically close the other field. Make sure to try this out with an active search query in both.

Reply:
Switched to everything being in one stack, and making sure the file opener closes when the search in file field is open and visa versa.